docs/data/material/discover-more/related-projects/related-projects.md

```diff
@@ -31,13 +31,15 @@ A set of reusable components for design tools is available, designed to match th
 
 ## IDE tools
 
-- [MUI Snippets](https://marketplace.visualstudio.com/items?itemName=vscodeshift.material-ui-snippets): VSCode extension providing snippets.
-- [MUI Codemorphs](https://marketplace.visualstudio.com/items?itemName=vscodeshift.material-ui-codemorphs): VSCode extension providing codemods.
+- [Material UI Snippets](https://marketplace.visualstudio.com/items?itemName=vscodeshift.material-ui-snippets): VSCode extension providing snippets.
+- [Material UI Codemorphs](https://marketplace.visualstudio.com/items?itemName=vscodeshift.material-ui-codemorphs): VSCode extension providing codemods.
 - [eslint: detect unused classes](https://github.com/jens-ox/eslint-plugin-material-ui-unused-classes): ESLint plugin to detect unused styling classes with `@mui/styles`.
 
 ## Components
 
-This is a collection of third-party projects that extend MUI.
+This is a collection of third-party projects that extend Material UI.
+
+> Note that some of these projects didn't update to support Material UI's latest version. Make sure you check that if any error comes up when trying to use them.
```

Comment:
I wonder though if we shouldn't actually remove the outdated projects instead of only warning folks about it 🤔

Reply:
As with the Learn page that we just updated, I think we should remove anything that isn't on v5.